api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/freemarker/FreemarkerHttpServlet.java

```diff
@@ -469,6 +476,9 @@ protected Map<String, Object> getPageTemplateValues(VitroRequest vreq) {
 
         // Add these accumulator objects. They will collect tags so the template can write them
         // at the appropriate location.
+
+        map.putAll(getCustomCss());
```

Comment:
I think instead you could create a static String field initialized with the same code and updated by SiteStyleController.